src/Microsoft.VisualBasic/src/Microsoft/VisualBasic/CompilerServices/IDOBinder.vb

```diff
@@ -1050,6 +1050,15 @@ Namespace Microsoft.VisualBasic.CompilerServices
         End Function
     End Class
 
+    Public Delegate Function SiteDelegate0(ByVal site As CallSite, ByVal instance As Object) As Object
```

Comment:
That's a good question. @AlekseyTs is this something that needs to be exposed in the contract? It looks like this change is supposed to fix something related to reflection in .NET Native, so my guess is that this doesn't need to be exposed in the reference assembly.